FHIR 101 - Practical Guide.ipynb

```diff
@@ -1497,11 +1499,246 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "##### Fetch all resources types\n",
+    "Fetching all `StructureDefinitions` will give an overview of what types of resources are in a server."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hidden": true
+   },
+   "outputs": [],
+   "source": [
+    "fetch('http://hapi.fhir.org/baseR4/StructureDefinition?_summary=count')\n",
+    "fetch('http://hapi.fhir.org/baseR4/StructureDefinition')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "#####  Fetch a resource's schema\n",
+    "The `$snapshot` parameter will return the full schema for a resource, including any `snapshot` that a resource inherits. \n",
+    "\n",
+    "```\n",
+    "# Example\n",
+    "GET [base]/StructureDefinition/[id]/$snapshot\n",
+    "```\n",
+    "\n",
+    "For example, say there is a resource that inherits from `Patient` and has a `StructureDefinition` that looks like this:\n",
+    "\n",
+    "```json\n",
+    "{\n",
+    "  \"resourceType\": \"StructureDefinition\",\n",
+    "  \"id\": \"591026\",\n",
+    "  \"name\": \"MyPatient\",\n",
+    "  \"type\": \"Patient\",\n",
+    "  \"baseDefinition\": \"http://hl7.org/fhir/StructureDefinition/Patient\",\n",
+    "  \"differential\": {\n",
+    "    \"element\": [ {\n",
+    "      \"id\": \"Patient.extension.race\",\n",
+    "      ...\n",
+    "    } ]\n",
+    "  }\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "Then `$snapshot` will return the full schema for this resource, combining the `differential` from above with the `snapshot` from `Patient`. This is useful because the `$snapshot` parameter will look at a resource's hierarchy and inheritance, and will merge it together, so there is no need for multiple API requests. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hidden": true
+   },
+   "outputs": [],
+   "source": [
+    "fetch('http://hapi.fhir.org/baseR4/StructureDefinition/591026/$snapshot')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "##### Examining a specific instance of a resource\n",
+    "The general format for looking at a specific instance is:\n",
+    "\n",
+    "`GET [base]/[type]/[id]`\n",
+    "\n",
+    "This will return the full payload for a resource with a specific ID."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hidden": true
+   },
+   "outputs": [],
+   "source": [
+    "fetch('http://hapi.fhir.org/baseR4/Observation/1286681')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "##### References\n",
+    "📘 For more information on references, refer to the [FHIR docs section](https://www.hl7.org/fhir/search.html#reference)\n",
```

Comment:
"FHIR docs section" sounds quite general like referring to the entire FHIR documentation. What about "refer to this section" or "refer to the reference search section"?